### PR TITLE
find keyword in pagetitle assessment

### LINKED ISSUE #340: create assessment for pageTitleKeywords
Create assessment for pageTitleKeywords

### PR DESCRIPTION
**Pull request by terw-dan**
This introduces the assessment for the find keyword in pagetitle research. 
Fixes #340

### CODE REVIEW

**Comment by andizer** [comment]
CR done.

File: spec/assessments/keywordPageTitleAssessmentSpec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+var pageTitleKeywordAssessment = require( "../../js/assessments/pagetitleKeyword.js" );
+var Paper = require( "../../js/values/Paper.js" );
+var Factory = require( "../helpers/factory.js" );
+var i18n = Factory.buildJed();
+
+describe( "an assessment to check if the keyword is in the pageTitle", function(){
+	it( "returns an assementresult with keyword not found", function(){
+		var paper = new Paper( "", {
+			keyword: "keyword",
+			title: "this is a title"
```
Comment (atimmer) [inline spec/assessments/keywordPageTitleAssessmentSpec.js]: You don't have to have a title here.

**Comment by atimmer** [comment]
It's broken.

File: js/analyzescorer.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ var assessments = {};
 assessments.wordCount = require( "./assessments/countWords.js" );
 assessments.fleschReading = require( "./assessments/calculateFleschReading.js" );
 assessments.linkCount = require( "./assessments/getLinkStatistics.js" );
-assessments.pageTitleKeyword = require( "./assessments/pageTitleKeyword.js" );
+assessments.pageTitleKeyword = require( "./assessments/pagetitleKeyword.js" );
```
Comment (jcomack) [inline js/analyzescorer.js]: If we are to follow proper camel case practices, the file should be renamed and not the reference to the file.
`pageTitleKeyword` would be the proper way of naming this file.

**Comment by jcomack** [comment]
CR 2 done

File: js/assessments/pagetitleKeyword.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ * Executes the pagetitle keyword assessment and returns an assessment result.
+ * @param {Paper} paper The Paper object to assess.
+ * @param {Researcher} researcher The Researcher object containing all available researches.
+ * @param {object} i18n The locale object.
+ * @returns {AssessmentResult} The result of the assessment with text and score
+ */
+var pageTitleKeywordAssessment = function( paper, researcher, i18n ) {
+	var findPageTitleKeywordResult = researcher.getResearch( "findKeywordInPageTitle" );
```
Comment (atimmer) [inline js/assessments/pagetitleKeyword.js]: Please rename this to `keywords`, because that's what this function returns.

**Comment by atimmer** [comment]
CR Done 3 👍🏻

**Pull request merged**